Lesson 2/DemoApp/app/src/main/java/mff/mdp/demoapp/data/GithubRepository.kt

```diff
@@ -0,0 +1,13 @@
+package com.gendigital.mff.lecture3.data
```

Comment:
Copy&Paste error, please fix the package names, I prefer to use the "com.gendigital....` package. Please check also other classes

Reply:
It's worse 😆 I pushed completely different project as well